app/lib/katello/concerns/base_template_scope_extensions.rb

```diff
@@ -35,6 +40,22 @@ def host_products(host)
         host_subscription_facet(host)&.installed_products
       end
 
+      def host_products_names(host)
+        if products = host_products(host)
+          products.map(&:name).join(', ')
+        else
+          ''
+        end
+      end
+
+      def host_collections(host)
+        host.host_collections
+      end
+
+      def host_collections_names(host)
+        host_collections(host).map(&:name).join(", ")
```

Comment:
here you don't have to use the macro from above, inside of this method, I think using `host.host_collections` would make more sense